### DISCUSSION TITLE
The APP ID - does it save anywhere in a log that we can access if we didn't save it somewhere at the process start?

### DISCUSSION THREAD

**Discussion by caressgents** [Pythagora-io/gpt-pilot#368] [Q&A]
Maybe a silly question, but is there a log somewhere to show past project APP ID's? I am balls deep in a project, and accidentally CTRL+C out and did not jot the App ID down... Any advice here or am I simply overlooking an easy solution to get back in where it left off?

**Comment by caressgents** [comment]
Closing - found debug.log - DUH.

**Comment by cdtenzi** [comment]
I have found a solution for this:

Inside the /pilot folder, there's the .env file. There you can find the database_name key, which contains either your SQLite file name or the postgreSQL database name.

Then, inside the app table, you will find all the apps you created with their corresponding IDs
